src/os/bluestore/BlueFS.cc

```diff
@@ -2179,7 +2264,7 @@ int BlueFS::readdir(const string& dirname, vector<string> *ls)
 {
   std::lock_guard<std::mutex> l(lock);
   dout(10) << __func__ << " " << dirname << dendl;
-  if (dirname.empty()) {
+  if (dirname.size() == 0) {
```

Comment:
empty() or size() both are constant complexity. Anything specific in using size()?